packages/compose/src/hooks/use-focus-outside/index.ts

```diff
@@ -32,6 +32,9 @@ type FocusNormalizedButton =
 function isFocusNormalizedButton(
 	eventTarget: EventTarget
 ): eventTarget is FocusNormalizedButton {
+	if ( eventTarget instanceof window.SVGElement ) {
```

Comment:
So the SVG is capturing the click instead of the button?

Reply:
Why is the SVG the target? If there's no other fix, can we generalise this to normalise for any element within a button? Also we should avoid using globals and use `ownerDocument.defaultView` intstead.